homeassistant/const.py

```diff
@@ -343,6 +343,14 @@
 TEMP_CELSIUS = "°C"
 TEMP_FAHRENHEIT = "°F"
 
+# Time units
```

Comment:
```diff
+TIME_MILLISECONDS = "ms"
```
...and suspects: `git grep -E '\<millis|\<ms\>'`

I had locally also tentatively
```diff
+TIME_MICROSECONDS = "μs"
```
...and a TODO note for TIME_WEEKS, although I'm not sure if either of these have any occurrences in the tree.

Reply:
Added in https://github.com/home-assistant/home-assistant/pull/32065/commits/524ee4855099d116fdafa961909aa437fde0ed0d